### ISSUE TITLE
Mysterious rm at the end of unit-tests all rule

### ISSUE THREAD

**Issue by sophiakoulen** [sophiakoulen/minishell#94]
When using make in unit-tests, it seems that binaries are automatically erased when we are finished running all the tests.
However, i can't find any rule expliciting this behaviour.
The consequence of this is that unit-tests are recompiling when using make all.

```
rm objs/quote_removal_test.o objs/tokenizer_test.o objs/parsing_test.o objs/param_expansion_test.o
```

Where is this coming from? Not from the clean rule, which i silenced!

**Comment by znichola** [comment]
liked with #23

**Comment by znichola** [comment]
fixed with #155
